Yoast/wordpress-seo
# Reduce bundle size

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Reduces javascript bundle size.

## Relevant technical choices:

* Added `@yoast/helpers` as a dependency to be able to output it as an external.
* Added `@yoast/analysis-report` as a dependency to be able to output it as an external.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn webpack-analyze-bundle`.
* Make sure `@yoast/search-metadata-previews` is no longer included in the post scraper, term scraper and search appearance bundle.
* Make sure the snippet preview still works as expected.

The following premium branch has these changes: `test-bundle-size`

## UI changes

## Quality assurance


Fixes Yoast/wordpress-seo-premium#2364

Requires https://github.com/Yoast/javascript/pull/267
